Controller/CRUDController.php

```diff
@@ -125,9 +125,9 @@ public function listAction()
      *
      * @throws AccessDeniedException If access is not granted
      */
-    public function batchActionDelete(ProxyQueryInterface $query)
+    public function deleteBatchAction(ProxyQueryInterface $query)
```

Comment:
Renaming method is BC break for PHP class extension. I don't how this could be BC.

Reply:
Maybe you could write a wrapper method @OskarStark

Reply:
ofc I can, and I know, that this is a BC break, but I don't have time to work on this PR